satpy/tests/compositor_tests/test_viirs.py

```diff
@@ -311,21 +307,21 @@ def test_reflectance_corrector_abi(self):
         self.assertEqual(res.attrs['area'], area)
         self.assertEqual(res.attrs['ancillary_variables'], [])
         data = res.values
-        self.assertLess(abs(np.mean(data) - 29.907390988422513), 1e-10)
+        self.assertLess(abs(np.nanmean(data) - 26.00760944144745), 1e-10)
         self.assertEqual(data.shape, (5, 10))
-        unique = np.unique(data)
+        unique = np.unique(data[~np.isnan(data)])
```

Comment:
How reliable is `unique` with floats ?

Reply:
:man_shrugging: Not sure, this was here before and seems to work. It is probably not great but it works. I think @wroberts4 was just trying to come up with something that checked the results without having to have the whole expected array output. Since the test data only have so many unique input values I don't think this is a super bad check even though it smells funny to do unique on floats.

Reply:
Ok. Don't blame me if it starts failing on some exotic platform some day :)

Reply:
Yeah, I won't be surprised and I won't blame you :smile: